src/sphinx_codeautolink/extension/block.py

```diff
@@ -70,9 +89,9 @@ def __init__(
         relative_path = Path(self.document['source']).relative_to(source_dir)
         self.current_document = str(relative_path.with_suffix(''))
         self.global_preface = global_preface
-        self.custom_blocks = {'pycon': clean_pycon}
+        self.custom_blocks = BUILTIN_BLOCKS
         self.custom_blocks.update(custom_blocks)
```

Comment:
```suggestion
        self.transformers = BUILTIN_BLOCKS.copy()
        self.transformers.update(custom_blocks)
```

I think the name is now outdated. Also I noticed that we've updated the original dict until now 😅 so let's be proper even though it wouldn't be a problem in practice. Of course, this would introduce changes below as well.

Reply:
Good catch!